src/main/java/PersonalBudget/business/expense/domain/model/ExpenseCategoryEntity.java

```diff
@@ -0,0 +1,36 @@
+package PersonalBudget.business.expense.domain.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Pay attention to the annotations you add and add  only those which are relevant. Getter and setter don't seem used in the project?  A golden rule is always write code which is needed now.